src/wazuh_db/wdb_parser.c

```diff
@@ -5470,22 +5470,26 @@ int wdb_parse_global_sync_agent_info_set(wdb_t * wdb, char * input, char * outpu
 }
 
 int wdb_parse_get_groups_integrity(wdb_t* wdb, char* input, char* output) {
+    if(strlen(input) < OS_SHA1_HEXDIGEST_SIZE){
+        merror("Error trying to get the hash hex-digest");
```

Comment:
I think this could be a DEBUG message, clarifying that the provided hash didn´t have the expected size.
The idea to turn it into a DEBUG message is because usually, these errors don´t require immediate user action.